src/CommandIsolator.cpp

```diff
@@ -48,6 +49,12 @@ class CommandIsolatorProcess : public process::Process<CommandIsolatorProcess> {
   }
 
  private:
+  inline static ::mesos::ResourceStatistics emptyStats() {
+    ::mesos::ResourceStatistics stats;
+    stats.set_timestamp(Clock::now().secs());
```

Comment:
I'd prefer to see timestamp passed as a parameter and you get clock::now().secs in the method usage